mesh/amqp.h

```diff
@@ -24,12 +24,12 @@ struct mesh_amqp;
 struct mesh_amqp *mesh_amqp_new(void);
 void mesh_amqp_free(struct mesh_amqp *amqp);
 
-
-
 const char *mesh_amqp_get_url(struct mesh_amqp *amqp);
 bool mesh_amqp_set_url(struct mesh_amqp *amqp, const char *url);
 const char *mesh_amqp_get_exchange(struct mesh_amqp *amqp);
 bool mesh_amqp_set_exchange(struct mesh_amqp *amqp, const char *exchange);
 const char *mesh_amqp_get_routing_key(struct mesh_amqp *amqp);
 bool mesh_amqp_set_routing_key(struct mesh_amqp *amqp, const char *routing_key);
-void mesh_amqp_publish(struct mesh_amqp *amqp, const void *data, size_t size, const char *routing_key);
+void mesh_amqp_publish(struct mesh_amqp *amqp, const void *data, size_t size);
+void mesh_amqp_close(struct mesh_amqp *amqp);
+void mesh_amqp_start_thread(struct mesh_amqp *amqp);
```

Comment:
Rename to `mesh_amqp_start`, using threads is somewhat embarrassing, let's not brag ;)